client/handle_http_test.go

```diff
@@ -738,128 +775,88 @@ func TestGetAndPutAuth(t *testing.T) {
 	// This tests object get/put with a pelican:// url
 	t.Run("testPelicanObjectPutAndGetWithPelicanUrl", func(t *testing.T) {
 		config.SetPreferredPrefix("pelican")
-		// Set path for object to upload/download
-		tempPath := tempFile.Name()
-		fileName := filepath.Base(tempPath)
-		uploadURL := "pelican:///test/" + fileName
-
-		// Upload the file with PUT
-		transferResultsUpload, err := DoPut(ctx, tempFile.Name(), uploadURL, false, WithTokenLocation(tempToken.Name()))
-		assert.NoError(t, err)
-		if err == nil {
-			assert.Equal(t, transferResultsUpload[0].TransferredBytes, int64(17))
-		}
-
-		// Download that same file with GET
-		transferResultsDownload, err := DoGet(ctx, uploadURL, t.TempDir(), false, WithTokenLocation(tempToken.Name()))
-		assert.NoError(t, err)
-		if err == nil {
-			assert.Equal(t, transferResultsDownload[0].TransferredBytes, transferResultsUpload[0].TransferredBytes)
-		}
-	})
-
-	// This tests pelican object get/put with an osdf url
-	t.Run("testPelicanObjectPutAndGetWithOSDFUrl", func(t *testing.T) {
-		config.SetPreferredPrefix("pelican")
-		// Set path for object to upload/download
-		tempPath := tempFile.Name()
-		fileName := filepath.Base(tempPath)
-		// Minimal fix of test as it is soon to be replaced
-		uploadURL := "pelican:///test/" + fileName
```

Comment:
@joereuss12, hoping for your feedback here. If you're already working on another branch that overhauls these tests and moves them to another file, I'm inclined not to spend more time trying to bring this back (especially since the `OSDFUrl` tests are the ones I'm struggling to get passing on MacOS)

Reply:
Personally, I think it is okay to remove for now. In this case, it is a duplicate anyway so it is not testing any additional tests. Brian's local cache PR moves all of these tests to a file called `fed_test.go` within the client so they should be safe there. Also my PR #878 changes these tests a good amount so if anything this is just a temporary loss of the tests

Reply:
Thanks for the update. I'll mark this as resolved for now. Sorry for all the merge conflicts this PR is going to cause you in your other PR!